pkg/apis/contrail/v1alpha1/vrouter_types.go

```diff
@@ -56,26 +56,19 @@ type VrouterSpec struct {
 // VrouterConfiguration is the Spec for the cassandras API.
 // +k8s:openapi-gen=true
 type VrouterConfiguration struct {
-	Containers          []*Container                `json:"containers,omitempty"`
-	ControlInstance     string                      `json:"controlInstance,omitempty"`
-	CassandraInstance   string                      `json:"cassandraInstance,omitempty"`
-	Gateway             string                      `json:"gateway,omitempty"`
-	PhysicalInterface   string                      `json:"physicalInterface,omitempty"`
-	MetaDataSecret      string                      `json:"metaDataSecret,omitempty"`
-	NodeManager         *bool                       `json:"nodeManager,omitempty"`
-	Distribution        *Distribution               `json:"distribution,omitempty"`
-	ServiceAccount      string                      `json:"serviceAccount,omitempty"`
-	ClusterRole         string                      `json:"clusterRole,omitempty"`
-	ClusterRoleBinding  string                      `json:"clusterRoleBinding,omitempty"`
-	VrouterEncryption   bool                        `json:"vrouterEncryption,omitempty"`
-	ContrailStatusImage string                      `json:"contrailStatusImage,omitempty"`
-	StaticConfiguration *VrouterStaticConfiguration `json:"staticConfiguration,omitempty"`
-}
-
-// VrouterStaticConfiguration stores static configuration of services
-// required by Vrouter
-// +k8s:openapi-gen=true
-type VrouterStaticConfiguration struct {
+	Containers                []*Container                 `json:"containers,omitempty"`
+	ControlInstance           string                       `json:"controlInstance,omitempty"`
```

Comment:
Are those fields "ControlInstanc" and "Cassandra" going to be used in vrouter controller?

Reply:
In vRouter controller it's not used. I was thinking on dropping them, but in Manager we may declare multiple cassandras and multiple controls resources so we cannot automatically assign proper cassandra and control to vrouter as we cannot say which vrouter should use which cassandra.

Reply:
I think that this field should stay in manager, but shouldn't be propagated to vrouter. Maybe it is time for a new version of API.
In other words. Manager struct should stay the same, however we may change vrouter type

Reply:
The same is for kubemanager.

Reply:
Moreover CassandraInstance in Vrouter looks totally unused in Vrouter and Manager

Reply:
Ok, I redesigned the API